testing/benchmark/main.tf

```diff
@@ -9,40 +9,34 @@ terraform {
       source  = "hashicorp/aws"
       version = ">=4.17.1"
     }
+    time = {
+      source  = "hashicorp/time"
+      version = ">=0.9.1"
+    }
   }
 }
 
+resource "time_static" "created_date" {}
+
 locals {
   ci_tags = {
-    environment  = coalesce(var.ENVIRONMENT, "dev")
-    repo         = coalesce(var.REPO, "apm-server")
+    environment  = var.ENVIRONMENT
+    repo         = var.REPO
     branch       = var.BRANCH
     build        = var.BUILD_ID
-    created_date = var.CREATED_DATE
-  }
-
-  mandatory_tags = {
-    division  = "engineering"
-    org       = "obs"
-    team      = "apm-server"
-    project   = "benchmarks"
+    created_date = coalesce(var.CREATED_DATE, time_static.created_date.unix)
   }
 }
 
 module "tags" {
-  source  = "../infra/terraform/modules/tags"
-  project = "benchmarks"
+  source = "../infra/terraform/modules/tags"
+  # use the convention for team/shared owned resources if we are running in CI.
+  # assume this is an individually owned resource otherwise.
+  project = startswith(var.user_name, "benchci") ? "benchmarks" : var.user_name
 }
 
 provider "ec" {}
 
-provider "aws" {
-  region = var.worker_region
-  default_tags {
-    tags = merge(local.ci_tags, module.tags.tags)
-  }
-}
```

Comment:
Did you mean to remove the entire `provider` block? AFAIK we still need it so we can specify the worker region. Before we merge, can you make sure trigger the benchmarks and that they pass? It should also be a good way to see the exact tags that are applied.

Reply:
Good catch! I've added back the block.

I've also confirmed that the terraform script are working and the benchmarks run :+1: